docker-compose-observable.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.4'
+services:
+
+## Observability
+
+  loki:
+    image: grafana/loki:1.5.0
+    ports:
+      - "3100:3100"
+    command: -config.file=/etc/loki/local-config.yaml
+    network_mode: host
+
+  prometheus:
+    image: prom/prometheus:v2.1.0
+    ports:
+      - 9090:9090
+    command:
+      - --config.file=/etc/prometheus/prometheus.yml
+    network_mode: host
+    volumes:
+      - ${HOST_WORKING_DIR}/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
+    environment:
+      HOST_WORKING_DIR: "${PWD}"
+
+  grafana:
+    depends_on:
+      - loki
+      - prometheus
+    image: grafana/grafana:master
+    ports:
+      - "3000:3000"
+    network_mode: host
+    volumes:
+      - ${HOST_WORKING_DIR}/grafana/datasources.yaml:/etc/grafana/provisioning/datasources/datasources.yaml
```

Comment:
Can't this just be `./grafana/datasources.yaml...?

Reply:
The paths have to be absolute paths. And the absolute path also has to work when the service containers are started from inside the "java-development" container. So, had to pass the absolute path to the "java-development" container.